returnn/datasets/postprocessing.py

```diff
@@ -228,10 +252,47 @@ def _make_tensor_template_from_input(self, data_key: str) -> Tensor:
         return Tensor(data_key, dims=dims, dtype=dtype, sparse_dim=sparse_dim)
 
 
+T = TypeVar("T", TensorDict, Iterator[TensorDict])
+
+
+def compose(*postprocessing_funcs: Callable):
```

Comment:
It's inconsistent that this is here a function (`compose`) vs having a class (`LaplaceOrdering`) otherwise.
I would suggest to make this here a class as well.